build.sbt

```diff
@@ -25,10 +25,13 @@ developers := List(
 
 publishTo := sonatypePublishToBundle.value
 
+val zioAwsVersion = "3.15.19.10"
```

Comment:
I realize a good aspect of that is I won't receive a Scala Steward PR daily to upgrade the AWS library 😆

Reply:
There might be those for zio-aws itself. I am please with `mergify` for automatically merging those PRs, some other zio projects are using it too.